OpenSim/Simulation/Test/testInitState.cpp

```diff
@@ -50,6 +50,7 @@ int main()
     try {
         LoadOpenSimLibrary("osimActuators");
         testStates("arm26.osim");
+        //TODO: testMemoryUsage fails, is that okay?
```

Comment:
Try rerunning the test a few times; does the output vary? The test should not produce that kinda output on the master branch.